packages/db/tests/lib/test-granule.js

```diff
@@ -728,6 +728,53 @@ test.serial('getGranulesByApiPropertiesQuery returns correct granules by provide
   );
 });
 
+test.serial('getGranulesByApiPropertiesQuery returns results POSIX/ASCII sorted when collition is set to "C"', async (t) => {
```

Comment:
Was the collate behavior present in the previous ES implementation?

Reply:
Not explicitly, but implicitly because of the default methods ES uses to sort - that's why the tests all broke due to sort order when uppercase/lowercase granuleIds are involved.